decentraland/unity-renderer
# fix: attached entities follow instant the player

## What does this PR change?

This PR fixes #449.

The attached entities follow the player in first person with a delay, now the entities follow the player instantly without a delay

## How to test the changes?


1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/attached-entities&position=-133%2C-40&ENV=org
2. Pick up the portal gun
3. See if it follows you correctly
4. Change between 1º and 3º person camera to check that everything work as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
